pkg/api/handlers.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/weaveworks/common/instrument"
 	"github.com/weaveworks/common/middleware"
 
-	"github.com/cortexproject/cortex/pkg/chunk/purger"
+	"github.com/cortexproject/cortex/pkg/storage/purger"
```

Comment:
Why not have a top level directory for purger? 
`pkg/purger`

Reply:
Yeah, that's a good option, I've moved it now.